app/js/lib/foxbox/services.js

```diff
@@ -67,8 +66,11 @@ export default class Services extends EventDispatcher {
     this[p.settings] = settings;
 
     this[p.cache] = null;
-    this[p.isPollingEnabled] = false;
-    this[p.nextPollTimeout] = null;
+    this[p.pollingTimer] = new SequentialTimer(
+      this[p.settings].pollingInterval
```

Comment:
In this case, you used `settings` directly instead of `this[p.settings]` in one of the files above.
Decide which one prefer and use it consistently.